fvtr/gdb/gdb.exp

```diff
@@ -27,7 +27,7 @@ if { [array names env -exact "AT_GDB_VER"] == "" } {
 # AT 5.0 and 6.0 used to use development versions of GDB, instead of stable
 # versions like AT >= 7.0.  Which causes issues when trying to run this
 # testcase, like when testing for GDB version.
-if { [regexp "5\..*|6\..*" $env(AT_MAJOR_VERSION)] } {
+if { [regexp "\A5\..*|\A6\..*" $env(AT_MAJOR_VERSION)] } {
```

Comment:
Completely subjective and perhaps personal preference, but would `^` work here instead of `\A`? The caret (`^`) is more familiar to me, at least.
(Same for the next change, as well.)

Reply:
@ThinkOpenly It does, but apparently requires an extra switch (`-lineanchor`). But I agree that is more standard and can make it less confusing to future us/others taking a look at this code.

But since tests are already running (and they take a long time), I'll leave that to a future PR addressing #1813.